eng/config/OptProf.json

```diff
@@ -18,20 +18,16 @@
               "filename": "/Microsoft.CodeAnalysis.EditorFeatures.Text.dll",
               "testCases":[ "Microsoft.Test.Performance.XamlOptProfCreateTests.WpfCreateProject_DesignerIsolated" ]
             },
-            {
-              "filename": "/DesktopHost/Microsoft.CodeAnalysis.Features.dll",
-              "testCases":[ "Microsoft.Test.Performance.XamlOptProfCreateTests.WpfCreateProject_DesignerIsolated" ]
-            },
             {
               "filename": "/Microsoft.VisualStudio.LanguageServices.dll",
               "testCases":[ "Microsoft.Test.Performance.XamlOptProfCreateTests.WpfCreateProject_DesignerIsolated" ]
             },
             {
-              "filename": "/DesktopHost/System.Reflection.Metadata.dll",
+              "filename": "/InteractiveHost/Desktop/System.Reflection.Metadata.dll",
```

Comment:
I assume we want to do a test insertion of this change to verify we are not breaking optprof

Reply:
Test insertion currently does not work for this branch. @genlu

Reply:
I believe we are snapping today, so this should be retargeting to master then.